<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>381</width>
     <height>240</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QSlider" name="sld_center">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>min. value</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>sld_min_value</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Center</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>sld_center</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSlider" name="sld_min_value">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Window</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>sld_window</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="spb_center">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>alpha</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>sld_alpha</cstring>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSlider" name="sld_alpha">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="sld_window">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="spb_window">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSlider" name="sld_max_value">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="spb_min_value">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QSpinBox" name="spb_max_value">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QSpinBox" name="spb_alpha">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>max. value</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>sld_max_value</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="lb_overlay">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Overlay</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="3">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sld_center</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spb_center</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spb_center</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sld_center</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sld_window</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spb_window</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spb_window</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sld_window</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sld_min_value</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spb_min_value</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spb_min_value</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sld_min_value</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sld_max_value</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spb_max_value</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spb_max_value</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sld_max_value</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sld_alpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spb_alpha</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spb_alpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sld_alpha</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
